src/main/java/bookstore/service/BookServiceImpl.java

```diff
@@ -16,6 +18,7 @@
 public class BookServiceImpl implements BookService {
     private final BookRepository bookRepository;
     private final BookMapper bookMapper;
+    private final CategoryRepository categoryRepository;
 
     @Override
     public BookDto save(CreateBookRequestDto requestDto) {
```

Comment:
can you save the book with categories? Let’s  modify the save  and update methods

Reply:
not fixed